### PR TITLE
Add /dine and /dineinfo features

### LINKED ISSUE #61: Food dining options feature
1. list all food places in a faculty
2. view specific location and opening hours of a food place

### PR DESCRIPTION
**Pull request by mrwsy1**
Add food dining options feature

### CODE REVIEW

File: src/main/java/seedu/duke/logic/commands/DineCommand.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                    } else {
+                        System.out.println("The dining options available at " + tempFaculty + " are:");
+                        for (DiningOptions info : foodPlaceInfo) {
+                            displayCount++;
+                            System.out.println(displayCount + ". " + info.getName());
+                        }
+                        break;
+                    }
+                }
+            }
```
Comment (wamikamalik) [inline src/main/java/seedu/duke/logic/commands/DineCommand.java]: Can this be refactored into another function or maybe a function in Ui?

Comment (mrwsy1) [reply src/main/java/seedu/duke/logic/commands/DineCommand.java]: i'll try that

Comment (wamikamalik) [reply src/main/java/seedu/duke/logic/commands/DineCommand.java]: Okie!

File: src/main/java/seedu/duke/logic/commands/DineCommand.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    public int printDineResult(int displayCount, String tempFaculty, ArrayList<DiningOptions> foodPlaceInfo) {
+        System.out.println("The dining options available at " + tempFaculty + " are:");
+        for (DiningOptions info : foodPlaceInfo) {
+            displayCount++;
+            System.out.println(displayCount + ". " + info.getName());
+        }
+        return displayCount;
+    }
```
Comment (yuxinwu99) [inline src/main/java/seedu/duke/logic/commands/DineCommand.java]: might be better to move this to UI

Comment (mrwsy1) [reply src/main/java/seedu/duke/logic/commands/DineCommand.java]: ok

**Review by wamikamalik — APPROVED** [review]
LGTM!

**Review by yuxinwu99 — APPROVED** [review]

**Pull request merged**